examples/shared.php

```diff
@@ -18,7 +18,7 @@ function printObject(ASNObject $object, $depth = 0)
 
     $content = $object->getContent();
     if (is_array($content)) {
-        foreach ($object as $child) {
+        foreach ($content as $child) {
```

Comment:
Both variants work because objects that contain array content implement the corresponding interface to allow ranging over all contained items.

Reply:
I tested it works as expected by executing one of the examples that use this function:

```shell
$ php examples/Issue57.php
…
```

I also found this unit test that tests this is working as expected:

```shell
$ grep -r Iterate tests 
tests/ASN1/Universal/SequenceTest.php:    public function testIterateSequence()
```